src/descriptors/power_system_structs.json

```diff
@@ -1418,6 +1418,121 @@
       ],
       "supertype": "StaticLoad"
     },
+    {
+      "struct_name": "ExponentialLoad",
+      "docstring": "Data structure for a static exponential load.",
+      "fields": [
+        {
+          "null_value": "init",
+          "name": "name",
+          "exclude_setter": true,
+          "data_type": "String"
+        },
+        {
+          "null_value": "false",
+          "name": "available",
+          "data_type": "Bool"
+        },
+        {
+          "name": "bus",
+          "null_value": "Bus(nothing)",
+          "data_type": "Bus"
+        },
+        {
+          "name": "active_power",
+          "null_value": "0.0",
+          "data_type": "Float64",
+          "needs_conversion": true
+        },
+        {
+          "name": "reactive_power",
+          "null_value": "0.0",
+          "data_type": "Float64",
+          "needs_conversion": true
+        },
+        {
+          "name": "active_power_coefficient",
+          "comment": "Coefficient relating voltage dependence for power P = P0 * V^α",
+          "null_value": "0.0",
+          "data_type": "Float64",
+          "valid_range": {
+            "min": 0,
+            "max": null
+          },
+          "validation_action": "warn"
+        },
+        {
+          "name": "reactive_power_coefficient",
+          "comment": "Coefficient relating voltage dependence for power Q = Q0 * V^β",
+          "null_value": "0.0",
+          "data_type": "Float64",
+          "valid_range": {
+            "min": 0,
+            "max": null
+          },
+          "validation_action": "warn"
+        },
+        {
+          "name": "base_power",
+          "comment": "Base power of the unit in MVA",
+          "null_value": "0.0",
+          "data_type": "Float64",
+          "valid_range": {
+            "min": 0,
+            "max": null
+          },
+          "validation_action": "warn"
+        },
+        {
+          "name": "max_active_power",
+          "null_value": "0.0",
+          "data_type": "Float64",
+          "needs_conversion": true
+        },
+        {
+          "name": "max_reactive_power",
+          "null_value": "0.0",
+          "data_type": "Float64",
+          "needs_conversion": true
+        },
+        {
+          "name": "services",
+          "data_type": "Vector{Service}",
+          "comment": "Services that this device contributes to",
+          "null_value": "Device[]",
+          "default": "Device[]"
+        },
+        {
+          "name": "dynamic_injector",
```

Comment:
Do we need a dynamic injector here?

Reply:
Probably for consistency. In the future when implementing the DynamicExponentialLoad we probably should attach it to an ExponentialLoad, but it is likely that any `StaticLoad` can hold any dynamic load.

So in the future, all dynamic loads should be attached to `PowerLoad` if we remove dynamic_injector of this struct.